BLUESPAWN-win-client/headers/util/wrappers.hpp

```diff
@@ -10,43 +10,41 @@
 template<class T>
 class GenericWrapper {
 protected:
-	std::shared_ptr<void> ReferenceCounter;
-
-	T WrappedObject;
+	std::shared_ptr<T> ReferenceCounter;
 	std::optional<T> BadValue;
 
 public:
 
-	GenericWrapper(T object, std::function<void(T)> freeFunction = [](T object){ delete object; }, std::optional<T> BadValue = std::nullopt) : 
-		WrappedObject{ object }, 
+	GenericWrapper(T object, std::function<void(T)> freeFunction = [](T object){ delete object; }, std::optional<T> BadValue = std::nullopt) :
```

Comment:
Codacy found an issue: [Class 'GenericWrapper < HandleWrapper >' has a constructor with 1 argument that is not explicit.](https://app.codacy.com/manual/ION28/BLUESPAWN/pullRequest?prid=5982821)